docs/Reference/API-Methods.md

```diff
@@ -4386,12 +4386,12 @@ None
 
 ### trace_replayBlockTransactions
 
-Provides transaction processing tracing.
+Provides transaction processing tracing per block.
 
 !!! important
 
     Your node must be an archive node (that is, synchronised without pruning or fast sync) or the
-    requested block must be within the last 1024 blocks.
+    requested block must be within [the number of pruning blocks retained](../CLI/CLI-Syntax#pruning-blocks-retained).
```

Comment:
Line length?

Reply:
It's a link putting it over so I'm going to leave as is